### DISCUSSION TITLE
Cannot build the Block project

### DISCUSSION THREAD

**Discussion by eherozhao** [microsoft/AirSim#3612] [Support Q&A] [answered]
I cannot use the VS 2017 to build the Block Project.

**Comment by rajat2004** [comment]
It's best to restore the original problem, and answer it with how you fixed it and select your reply as the answer. That'll be useful for anyone facing similar problem and coming across this question

**Reply by eherozhao** [reply]
Sure thing. I will update it. Thank you.

**Comment by eherozhao** [comment, marked as answer]
I installed the VS 2019 and it works perfectly. I am interested in why VS 2017 didn't work.
